app.mjs

```diff
@@ -19,7 +19,7 @@ const chainIdParamSchema = {
     properties: {
       chainId: {
         type: "number",
-        enum: [1, 250],
+        enum: [1, 250, 42161],
```

Comment:
Some extra commenting on which chain each ID refers to would be great

Reply:
Maybe turn them into constants?

Reply:
Done, let me know if you'd prefer the constants in a different format/structure